service/deployer/installer/installer.go

```diff
@@ -53,13 +55,31 @@ func New(config Config) (spec.Installer, error) {
 
 	var err error
 
+	var configurerService configurerspec.Configurer
+	{
+		configurerConfig := configurer.DefaultConfig()
+
+		configurerConfig.Logger = config.Logger
+
+		configurerConfig.Flag = config.Flag
+		configurerConfig.Viper = config.Viper
+
+		configurerConfig.Type = configurerspec.ConfigurerType(config.Viper.GetString(config.Flag.Service.Deployer.Installer.Configurer.Type))
```

Comment:
Super small nitpick: Line feels super long, maybe split it into two to make it more readable?
Not sure if we do that / have guidelines for stuff like this.